clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -738,13 +738,61 @@ describe("Consent banner", () => {
       });
     });
 
+    describe("when experience is empty, and geolocation is not provided", () => {
+      beforeEach(() => {
+        stubConfig({
+          geolocation: OVERRIDE.UNDEFINED,
+          experience: OVERRIDE.EMPTY,
+          options: {
+            isGeolocationEnabled: true,
+            geolocationApiUrl: "https://some-geolocation-url.com",
+          },
+        });
+      });
+
+      it("does fetches geolocation and does not render the banner", () => {
+        // we still need geolocation because it is needed to save consent preference
+        cy.wait("@getGeolocation");
+        cy.get("div#fides-banner").should("not.exist");
+        cy.contains("button", "Accept Test").should("not.exist");
+      });
+      it("does not render modal link", () => {
+        cy.get("#fides-modal-link").should("not.be.visible");
+      });
+    });
+
+    describe("when experience is empty, and geolocation is provided", () => {
+      beforeEach(() => {
+        stubConfig({
+          geolocation: {
+            country: "US",
+            location: "US-CA",
+            region: "CA",
+          },
+          experience: OVERRIDE.EMPTY,
+          options: {
+            isGeolocationEnabled: true,
+            geolocationApiUrl: "https://some-geolocation-url.com",
+          },
+        });
+      });
+
+      it("does not geolocate and does not render the banner", () => {
```

Comment:
A slightly better test here would be to ensure that it still attempts to fetch privacy experience in this scenario.

Reply:
Actually I got this backwards, never mind!